src/components/CmsForm/CmsForm.module.scss

```diff
@@ -1,12 +1,12 @@
 .globalError {
-    background-color: #f00;
+    background-color: var(--error);
```

Comment:
Tady si mi to nelibi - error je nic nerikajici nazev promenny, nemela by to byt --bg-error? Asi bude jina barva pro --text-error nez --bg-error, ne?
Pak jen drobnost, zda tomu chceme rikat error nebo casteji pouzivane alert/danger